lib/transport/transport-udp-socket.c

```diff
@@ -185,7 +185,7 @@ log_transport_udp_setup_fd(LogTransportUDP *self, gint fd)
   gint domain;
   socklen_t domain_len = sizeof(domain);
 
-  self->bind_addr = g_sockaddr_new_from_local_name(fd);
+  self->bind_addr = g_socket_get_local_name(fd);
```

Comment:
I got the following warning for implicit function declaration. Probably the header is missing somewhere.

```
/home/furiel/workspace/destip-destport-proto-macros/syslog-ng-core/lib/transport/transport-udp-socket.c:188:21: warning: implicit declaration of function ‘g_socket_get_local_name’ [-Wimplicit-function-declaration]
   self->bind_addr = g_socket_get_local_name(fd);
```

Reply:
thanks, fixed.